packages/common/src/number.ts

```diff
@@ -19,3 +22,20 @@ export const formatNumber = (value: number, decimalPlaces: number) => {
 export const randomInt = (min: number, max: number) => {
   return Math.floor(Math.random() * (max - min + 1) + min);
 };
+
+/**
+ *  Number of bytes to si format. (division by 1024)
+ *  Does not accept floats, size in bytes are should be integer.
+ */
+export const humanFileSize = (size: number) => {
+  if (!Number.isInteger(size)) return NaN;
```

Comment:
Do we really want to return NaN when the input size is not an integer?

Reply:
As discussed on discord:
I need something that somehow show an error on miss-use.
This function should only be used with whole numbers, or integers.
In case the dev is giving a float, it would show quickly that the value entered does not fit the format of "number of bytes".
Potential options:
 - Return "NaI" as string, so that the function's output type is always string.
 - Throw error, seems a bit over the top and potentially will bother an actual user before a dev could catch it
 - Math.floor it, blaming the dev for miss-use and potentially letting the issue unseen forever.
 - Keeping number as it could alert the developer early by the output missmatch forcing to at least understand correctly what the params and output are.

Reply:
What about console.error and NaN?